requirements-test-libraries.txt

```diff
@@ -66,9 +66,10 @@ git+https://github.com/shotgunsoftware/python-api.git@v3.2.6
 spacy==3.2.0
 srsly==2.4.2
 swagger-spec-validator==2.7.4
+tableauhyperapi==0.0.13821

```

Comment:
Why is this entry needed?

Reply:
It was probably just moved to have the entries ordered alphabetically?

Reply:
Oh so it is. I didn't spot that it was a move rather than an addition.